src/App.js

```diff
@@ -1,24 +1,22 @@
-import logo from './logo.svg';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import BooksContainer from './components/BooksContainer';
+import Navbar from './components/Navbar';
+import BookForm from './components/BookForm';
+import Categories from './pages/Categories';
+
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<BooksContainer />} />
+        <Route path="categories" element={<Categories />} />
+      </Routes>
+      <BookForm />
```

Comment:
- [ ] Instead of calling `BookForm` from here. As per requirement, you have to call from `BooksContainer` component so that it will show only `/` path rather than showing in both `/` and `categories` path.